Wangscape/Options.h

```diff
@@ -17,13 +20,20 @@ class Options
     typedef std::vector<Clique> CliqueList;
 
     std::string filename;
+    std::string directory;
```

Comment:
A class instance in `Options` sounds like the best idea. The code to update `directory` can be delegated there.

However, I'm not sure it makes sense for `OptionsLocation`/`Options::Location` for it to be a nested class when `TileFormat` and similar classes are independent?